template/src/SDKAppWrapper.tsx

```diff
@@ -20,12 +20,18 @@ import {Unsubscribe} from 'nanoevents';
 //   p: Parameters<T>[0],
 // ) => ReturnType<T>;
 
+type deviceId = MediaDeviceInfo['deviceId'];
+
 export interface SdkMethodEvents {
   customize: (customization: CustomizationApiInterface) => void;
   join(
     roomid: string | Partial<MeetingInfoContextInterface['data']>,
     skipPrecall?: boolean,
   ): MeetingInfoContextInterface['data'];
+  // mediaDevice: (deviceId: string, kind: MediaDeviceInfo['kind']) => void;
+  microphoneDevice: (deviceId: deviceId) => void;
+  speakerDevice: (deviceId: deviceId) => void;
+  cameraDevice: (deviceId: deviceId) => void;
```

Comment:
Since this is no longer being used, move this interface to SdkMethodEvents.ts, also remove the commented code below